vagrant/ansible/ssh-config

```diff
@@ -0,0 +1,33 @@
+Host k8s-master
+  HostName 127.0.0.1
+  User vagrant
+  Port 2222
+  UserKnownHostsFile /dev/null
+  StrictHostKeyChecking no
+  PasswordAuthentication no
+  IdentityFile /home/bernard/.vagrant.d/insecure_private_key
```

Comment:
I think this should be configurable (also see below)

Reply:
oops. I think ssh-config and inventory will be auto generated, so I'll just rm from repo